jena-text/pom.xml

```diff
@@ -112,11 +138,72 @@
         <groupId>org.apache.maven.plugins</groupId>
         <artifactId>maven-surefire-plugin</artifactId>
         <configuration>
-          <includes>
-            <include>**/TS_*.java</include>
-          </includes>
+            <!-- Skip the default running of this plug-in (or everything is run twice...see below) -->
+            <skip>true</skip>
         </configuration>
+          <executions>
+              <execution>
+                  <id>unit-tests</id>
+                  <phase>test</phase>
+                  <goals>
+                      <goal>test</goal>
+                  </goals>
+                  <configuration>
+                      <skip>false</skip>
+                      <includes>
+                          <include>**/TS_*.java</include>
+                      </includes>
+                      <excludes>
+                          <exclude>**/*IT.java</exclude>
+                      </excludes>
+                  </configuration>
+              </execution>
+              <execution>
+                  <id>integration-tests</id>
+                  <phase>integration-test</phase>
+                  <goals>
+                      <goal>test</goal>
+                  </goals>
+                  <configuration>
+                      <skip>false</skip>
+                      <includes>
+                          <include>**/*IT.java</include>
```

Comment:
It would nice to collect up the ITs in test suites in the same way as the unit tests are collected, just to maintain a uniform method of work.